OsiguSDK.Providers/Clients/V1/DiagnosesClient.cs

```diff
@@ -15,18 +15,9 @@ public DiagnosesClient(IConfiguration configuration) : base(configuration)
         {
         }
 
-
-        public Product GetDiagnosis(string diagnosisId)
-        {
-            var urlBuilder = new StringBuilder("/v1/diagnoses/").Append(diagnosisId);
-            var requestData = new RequestData(urlBuilder.ToString(), Method.GET, null, null);
-            return ExecuteMethod<Product>(requestData);
-        }
-
-
-        public Pagination<Diagnosis> GetListOfDiagnoses()
+        public Pagination<Diagnosis> GetListOfDiagnoses(string diagnosisSearch)
```

Comment:
Is `DiagnosesClient` a public interface intended to be used by developers working on integrations with our API? If that's so, I'm not sure it's a good idea to break that interface with a change like this one here, it would cause de developer to change everywhere they use `GetListOfDiagnoses` to now provide a `diagnosis`. If I where one of them, I would hate osigu for introducing breaking changes like this.

Reply:
@dmarenco please we need to be backwards compatibility always, in this case we could proceed by adding a new method in the interface but not overwriting the old one